ecl/eclcc/eclcc.cpp

```diff
@@ -422,6 +422,7 @@ class EclCC
     bool optIgnoreCache = false;
     bool optIgnoreSimplified = false;
     bool optExtraStats = false;
+    bool optPruneArchive = true;
```

Comment:
Does defaulting to true change existing behaviour?

Reply:
Yes, it means that plugins and the standard library are not added to archives.  They are already replaced in the eclccserver, so this shouldn't have any noticable effect as far as the user is concerned.